Design rules/readme.md

```diff
@@ -0,0 +1,124 @@
+# Design Rules VNGRealisatie
+
+## Inleiding
+
+In de onderliggende Design Rules zijn de inzichten vastgelegd die zijn opgedaan bij het ontwerpen en specificeren van API's binnen diverse projecten waar VNG Realisatie bij betrokken is.
+Deze Design Rules zijn binnen het team Standaarden besproken en er is consensus bereikt over het feit dat deze Design Rules toegepast worden op API-specificaties die door VNR Realisatie worden gemaakt of in beheer worden genomen.
+
+Dit is een levende set van Design Rules die beïnvloed kan worden door voortschrijdend inzicht en door technologische ontwikkelingen.
+
+Om die reden kan het zijn dat de specificaties die door VNG Realisatie zijn opgesteld of in beheer zijn genomen niet voldoen aan alle Design Rules.
+Op het moment dat een nieuwe versie van een dergelijke  API-specificatie gepland wordt waarmee breaking changes worden doorgevoerd bepaalt de betreffende designer of dat een kans is om correcties door te voeren waardoor de API weer voldoet aan de onderstaande Design Rules.
+
```

Comment:
Voortschrijdend inzicht kan er ook voor zorgen dat we anders aan gaan kijken tegen een Design Rule. Misschien wordt wel duidlijk dat het beter is om de Design Rule van deze lijst af te voeren.

Reply:
Het lijkt me dan beter om bij die Design Rule aan te geven dat hij niet meer geldig is. Ook de datum per wanneer die niet meer geldig is opnemen. Bij oudere API-specs ben je anders een toegepaste Design Rule kwijt. Deze toelichting voeg ik toe.